src/main/scala/com/danielasfregola/twitter4s/entities/Webhook.scala

```diff
@@ -0,0 +1,3 @@
+package com.danielasfregola.twitter4s.entities
+
+final case class Webhook(id: String, url: String, valid: Boolean, created_timestamp: String) // TODO - convert to Instant? parsers doesn't support actual format
```

Comment:
About this TODO: you can write a custom parser to support whatever format they use.
We should probably be doing that before releasing this PR...if you do not have the time/do not know how to do it, I can have a look in a couple of weeks :)

Reply:
Yes you're right, I added a formatter to do so 🙂